spec/client_spec.rb

```diff
@@ -69,10 +69,10 @@
     end
   end
 
-  shared_examples 'an event sender' do
+  shared_examples 'an event sender' do |o|
```

Comment:
sure. this test suite is pretty crappy, so a refactor will be needed soon — I couldn't use `options` because it crashes in weird ways.